schematypes.go

```diff
@@ -20,12 +20,33 @@ package dap
 
 // Message is an interface that all DAP message types implement with pointer
 // receivers. It's not part of the protocol but is used to enforce static
-// typing in Go code.
+// typing in Go code and provide some common accessors.
 //
 // Note: the DAP type "Message" (which is used in the body of ErrorResponse)
 // is renamed to ErrorMessage to avoid collision with this interface.
 type Message interface {
-	isMessage()
+	GetSeq() int
+	GetType() MessageType
+}
+
+type MessageType int
+
+const (
+	RequestType = iota
```

Comment:
Might be a good idea to be explicit for readability:
RequestType MessageType = iota
Or is there some rule against that?

Reply:
Done